homeworks/max.snischuk_snischuk/PokemonProject/index.html

```diff
@@ -32,7 +32,7 @@
         </header>
         <main class="about">
             <div class="about__container">
-                <img class="about__image" src="./images/about.png" alt="pokemon">
+                <img class="about__image" src="./images/about.jpeg" alt="pokemon">
```

Comment:
As a recommendation: the language of your site is English, but you have the Ukrainian error messages. Looks inconsistent.
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4cc78eaf-c649-442b-adb0-89e293b76e0b)

Reply:
The height of the input is different for normal and focus states - it leads to `content-jumping`
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/f21c7000-5fd8-4647-93be-5ce5d26d1529)
![Screenshot at Dec 14 19-19-45](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/5c4cd906-c2a7-467f-945d-24e31dc4ad9b)